packages/4-xdapp-layer/solana/src/context.ts

```diff
@@ -794,4 +794,19 @@ export class SolanaContext
   parseRelayerPayload(payload: Buffer): ParsedRelayerPayload {
     throw new Error('relaying is not supported on solana');
   }
+
+  // Implement relayer methods, puke immediately
```

Comment:
Lol, assuming it threw a type error because these were missing... I'm certain there's a better way to handle this, don't worry, will make it better ☀️

Reply:
yea just added it to make it stop whining